soroban-env-host/src/budget.rs

```diff
@@ -143,6 +203,134 @@ impl CostType {
     }
 }
 
+pub const PARAMS_CPU: &'static [[u64; 5]; 60] = &[
```

Comment:
I think having this big table maintained in un-checked alignment with the CostType enum is a recipe for maintenance-induced errors. Can we change it so that you have a big function that matches on `CostType` and returns a `CostModel` in each arm of the match? Then you will get the match-expression exhaustiveness-checker checking that you cover all cases and that each case matches the thing you think it should (i.e. no need for all these comments to be here, much less be lined up correctly)

Reply:
(returns a `CostModel` or modifies an existing already-retrieved `CostModel` or whatever .. just some pattern that uses the static enumeration of `CostType`s to our advantage)